packages/styles/src/components/_popover.scss

```diff
@@ -0,0 +1,18 @@
+$base-class: 'popover';
+
+.#{$base-class} {
+  background-color: var(--surface-basic-default);
+  border-radius: 4px;
+  box-shadow: 0 1px 10px 0 rgba(66, 77, 87, 0.3);
+  display: none;
+  max-width: 336px;
+  min-width: 168px;
+
+  &:focus {
+    outline: none;
+  }
+
+  &__visible {
```

Comment:
`visible` is a some kind of state, not a nested element, so please use the modifier `--`.

Reply:
👌🏼